src/common/base-store.ts

```diff
@@ -172,16 +175,19 @@ export class BaseStore<T = any> extends Singleton {
     return subFrames;
   }
 
-  @action
-  protected fromStore(data: T) {
-    if (!data) return;
-    this.data = data;
```

Comment:
Are we really happy with this change.. does this mean that each store needs to decide where it sets data?